### ISSUE TITLE
No-unbound-method warn when tested 'toHaveBeenCalled' in test

### ISSUE THREAD

**Issue by VincentLanglet** [palantir/tslint#4500]
### Bug Report

- __TSLint version__: 5.12.0
- __TypeScript version__: 3.3.1
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
expect(localStorage.getItem).toHaveBeenLastCalledWith('token');
```

with `tslint.json` configuration:

```json
{
  "extends": ["tslint:all"]
}
```

#### Actual behavior
Avoid referencing unbound methods which may cause unintentional scoping of 'this'.

#### Expected behavior
No error

**Comment by JoshuaKGoldberg** [comment]
Good news, @VincentLanglet - duplicate of #3755, which was resolved by #4472!

**Comment by VincentLanglet** [comment]
@JoshuaKGoldberg Great !
Sorry for the duplicate
